polkadot/xcm/xcm-runtime-apis/Cargo.toml

```
@@ -1,6 +1,6 @@
 [package]
 name = "xcm-runtime-apis"
-version = "0.1.0"
+version = "0.2.0"
```

Since the API and crate are backwards compatible, let's only bump patch number so that we can backport this change.

Bumping minor or major doesn't allow backporting this to existing stable releases.

```suggestion
version = "0.1.1"
```